tests/components/whirlpool/test_sensor.py

```diff
@@ -90,6 +109,25 @@ async def test_washer_sensor_values(
     mock_sensor1_api: MagicMock,
 ):
     """Test the sensor value callbacks."""
+    hass.state = CoreState.not_running
+    thetimestamp: datetime = datetime(2022, 11, 29, 00, 00, 00, 00, timezone.utc)
```

Comment:
I fail to understand how the added tests verify the 60 second limits on update? Or are they not doing that?